app/browser/windows.js

```diff
@@ -61,6 +61,14 @@ const updateWindow = (windowId, updateDefault = false) => {
   }
 }
 
+const onWindowResize = (windowId) => {
+  const windowValue = getWindowValue(windowId)
+  if (windowValue) {
+    appActions.onWindowResize(windowValue)
+    windowActions.onWindowUpdate(windowId, true)
```

Comment:
btw just noting that appActions with a windowId get routed to that window.

Reply:
not sure that I follow, are you saying that I should add windowId to the app action and this way I don't need separate windowAction?

Reply:
or just pass it into the dispatched action ya and then handle the app action resize constant in the window store. It's a bit abnormal to use windowActions. I see them as more only within a single window renderer. I think it's only used 1 other place.

Reply:
done
+